seed/ts-sdk/exhaustive/multiple-exports/src/api/resources/inlinedRequests/exports.ts

```diff
@@ -0,0 +1,3 @@
+// This file was auto-generated by Fern from our API Definition.
+
+export * from "./client/index.js";
```

Comment:
Inside of `./client/index.js`, the client class is not exported, so you'll need to export it separately in exports.ts